### ISSUE TITLE
Firefox removes p tag after select all and delete

### ISSUE THREAD

**Issue by anem** [yabwe/medium-editor#956]
If you have several paragraphs of text and you select all, then delete, when you start typing again the first line is no longer contained in a p tag. When you resume typing, the first line is unwrapped and the next is correctly wrapped in a paragraph.

This only seems to be an issue in firefox 44.0 (Works as expected on Chrome 48.0, not yet tested in IE).

Attached a screenshot showing the result (snippet from FireBug HTML view).

settings used:
`var editor = new MediumEditor('.editable', {
	disableDoubleReturn: true,
	disableExtraSpaces: true,
	spellcheck: true,
	placeholder: { text:"Please type your entry here" },
	paste: {
		cleanPastedHTML: true,
		cleanReplacements: [[/\s\s+/g, ' ']]
	},
	toolbar: {
		buttons: [
			'bold',
			'italic', 
			'underline',
		]
	}
});`

![screen shot 2016-02-08 at 10 31 26](https://cloud.githubusercontent.com/assets/79071/12883345/4dfa2830-ce4f-11e5-8bfc-2b02f82ee033.png)

**Comment by nmielnik** [comment]
Yeah, this is one of a few known issues around when various browsers choose to insert (or in this case not-insert) the `<p>` tags.

We would likely need something to be monitoring for changes and trying to detect that this happened, and then wrapping these dangling text nodes inside of a `<p>`.
